containers/support/AttachScreenshot.js

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react';
+import { c } from 'ttag';
+import { Text, Icon, Button, FileInput, useNotifications } from 'react-components';
+import { toBlob, resize } from 'proton-shared/lib/helpers/image';
+import { MAX_SIZE_SCREENSHOT } from 'proton-shared/lib/constants';
+import PropTypes from 'prop-types';
+
+const AttachScreenshot = ({ id, onUpload, onReset }) => {
+    const [attached, setAttached] = useState(false);
+    const { createNotification } = useNotifications();
+
+    const handleClick = () => {
+        setAttached(false);
+        onReset();
+    };
+
+    const handleChange = async ({ target }) => {
+        const images = [...target.files].filter(({ type }) => /^image\//i.test(type));
```

Comment:
idem should be a hlper

Reply:
This is a filter on a group of file to extract images. That's specific to this upload I think.